index.html

```diff
@@ -1,19 +1,249 @@
+                  <td>ReactJS</td>
+                </tr>
+                <tr>
+                  <td>NodeJS</td>
+                </tr>
+              </table>
+            </div>
+            <!-- END RIGHT -->
+          </div>
+          <!-- END ROW -->
+        </section>
+        <!-- END INTRO -->
+
+        <!-- WORK SECTION -->
+        <section id="work">
+          <h2>My Work History</h2>
+          <table>
+            <tr>
+              <th>Years</th>
+              <th>Company</th>
+              <th>Title</th>
+              <th class="no-mobile">Tech Used</th>
+            </tr>
+            <tr>
+              <td>1</td>
+              <td>Some Company</td>
+              <td>Jr Position</td>
+              <td class="no-mobile">HTML5</td>
+            </tr>
+            <tr>
+              <td>1.5</td>
+              <td>Another Company</td>
+              <td>More Important Position</td>
+              <td class="no-mobile">HTML5, CSS3, Javascript</td>
+            </tr>
+            <tr>
+              <td>3</td>
+              <td>A Big Company</td>
+              <td>Super Important Position</td>
+              <td class="no-mobile">ReactJS, NodeJS</td>
+            </tr>
+          </table>
+        </section>
+        <!-- END WORK SECTION -->
+
+        <!-- PORFOLIO SECTION -->
+        <section id="portfolio">
+          <h2>My Latest Projects</h2>
+          <!-- ROW: EACH FIGURE IS A COLUMN -->
+          <div class="row">
+            <figure>
+              <a href="#"
+                ><h3>Project Title</h3>
+                <img src="/images/pic05.jpg" alt="" />
+              </a>
+              <figcaption>
+                A recent project that looks and works great...
+              </figcaption>
+            </figure>
+            <figure>
+              <a href="#">
+                <h3>Project Title</h3>
+                <img src="/images/pic06.jpg" alt="" />
+              </a>
+              <figcaption>
+                A recent project that looks and works great...
+              </figcaption>
+            </figure>
+            <figure>
+              <a href="#">
+                <h3>Project Title</h3>
+                <img src="/images/pic07.jpg" alt="" />
+              </a>
+              <figcaption>
+                A recent project that looks and works great...
+              </figcaption>
+            </figure>
+          </div>
+          <!-- END ROW -->
+          <!-- ROW: EACH FIGURE IS A COLUMN -->
+          <div class="row">
+            <figure>
+              <a href="#">
+                <h3>Project Title</h3>
+                <img src="/images/pic02.jpg" alt="" />
+              </a>
+              <figcaption>
+                A recent project that looks and works great...
+              </figcaption>
+            </figure>
+            <figure>
+              <a href="#">
+                <h3>Project Title</h3>
+                <img src="/images/pic03.jpg" alt="" />
```

Comment:
on all the images I would remove the first slash since they will not load if you host your site, all your images don't show up when it's hosted such